doc/installing.md

```diff
@@ -60,4 +60,10 @@ $ make install
 ```
 The default installation prefix is `/usr/local`. Change this to your liking.
 
+Other cmake variables that maybe useful to set are
+* `BOOST_ROOT` path to the root directory of Boost
```

Comment:
Would make more sense around line 47, as it regards dependencies.

Reply:
Moved it earlier.